fcl/gen/corsika/prodcorsika_proton_intime_icarus_numi_sce_on.fcl

```diff
@@ -0,0 +1,3 @@
+#include "prodcorsika_proton_intime_icarus_numi.fcl"
+
+#include "enable_spacecharge_icarus.fcl"
```

Comment:
I am trying to move away from `sce_on` in favour of just `sce`; but since we are mimicking `prodcorsika_proton_intime_icarus_bnb_sce_on.fcl` here the name you gave makes more sense.